src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,34 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_info(self, type='object'):
```

Comment:
I agree `get_browser_info` is pretty vague. If I got it right, *desired capabilities* is a commonly understood term in the Selenium community, and this keyword could just be named `get_desired_capabilities`. I don't thing `browser` is needed there but wouldn't mind it either.